dbos_transact/logger.py

```diff
@@ -19,3 +42,33 @@ def config_logger(config: ConfigFile) -> None:
         )
         console_handler.setFormatter(console_formatter)
         dbos_logger.addHandler(console_handler)
+
+    otlp_logs_endpoint = (
+        config.get("telemetry", {}).get("OTLPExporter", {}).get("logsEndpoint")  # type: ignore
+    )
+    if otlp_logs_endpoint:
+        # Configure the DBOS logger to also log to the OTel endpoint.
+        log_provider = LoggerProvider(
+            Resource.create(
+                attributes={
+                    "service.name": "dbos-application",
+                }
+            )
+        )
+        set_logger_provider(log_provider)
+        log_provider.add_log_record_processor(
+            BatchLogRecordProcessor(OTLPLogExporter(endpoint=otlp_logs_endpoint))
+        )
+        otlp_handler = LoggingHandler(
+            level=logging.NOTSET, logger_provider=log_provider
```

Comment:
`NOSET` seems to be the default when not provided

https://opentelemetry-python.readthedocs.io/en/latest/_modules/opentelemetry/sdk/_logs/_internal.html#LoggingHandler

I assume the level is set dynamically by each call.